### PR TITLE
JUnit reporter does not output time #275

### PR DESCRIPTION
**Pull request by hugosenari**
Simple implementation, that works.

### CODE REVIEW

File: lib/reporters/junit.js [hunk 1]
```diff
@@ -108,13 +108,21 @@ exports.run = function (files, opts, callback) {
                 errorCount: 0,
                 failureCount: 0,
                 tests: 0,
-                testcases: [],
-                name: name
+                testcases: {},
```
Comment (mreinstein) [inline lib/reporters/junit.js]: @hugosenari sorry this took so long. Why converting from array to dictionary here?

Comment (hugosenari) [inline lib/reporters/junit.js]: Because is more easy to find them again later at testDone

**Comment by cdeveas** [comment]
Any hope for this being merged? It'd be useful for our legacy project which uses nodeunit on circle-ci.

**Comment by cdeveas** [comment]
Awesome! Could we get a publish, too?

**Pull request merged**